packages/cli/lib/base-generator.js

```diff
@@ -196,7 +196,7 @@ module.exports = class BaseGenerator extends Generator {
       return;
     }
     for (const o in opts) {
-      if (this.options[o] == null) {
+      if (!this.options[o]) {
```

Comment:
Ditto

Reply:
What happens when this.options[o] is actually an empty string? I've created a repo to reproduce the issue [here](https://github.com/aaqilniz/lb4-datasource-bug).
btw this change is also coming from the master branch. I created another PR with the master branch [here](https://github.com/loopbackio/loopback-next/pull/9014). I think I need to re-create the PRs by creating separate branches for each change.

Reply:
@achrinza Can you please confirm and guide me if I should be creating the PRs again and closing the existing ones? Then have a separate branch for every change. 

I have got a few more changes that I will be creating PRs for - so how should I proceed about this?